src/components/select/select.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useState } from 'react';
+import { SelectMain, SelectTitle, SelectWrapper } from './styles';
+import { SelectProps } from './types';
+
+export const Select: FC<SelectProps> = ({ data, onSelectChange }) => {
```

Comment:
better to name it as GenreSelect since you coupled it with the 'Genre' title. Or you can remove the title and provide it with props, with such an approach you will make a component more generic and more extendable.

You can go even further and set not just a text for the title as a prop but allow to add jsx for a title like this:

`
 <Select 
 data={selectOptions} 
 onSelectChange={onSelectChange}
 title={<SomeCustomeStyledTitle>title</SomeCustomeStyledTitle>} 
 />
`
and if it's not a jsx you can set your default SelectTitle
With such implementation, you will follow the _Open Closed_ principle from SOLID